src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysis.DisposeDataFlowOperationVisitor.cs

```diff
@@ -113,55 +113,34 @@ private void HandleDisposingOperation(IOperation disposingOperation, IOperation
                 }
             }
 
-            private void HandlePossibleEscapingOperation(IOperation escapingOperation, IOperation escapedInstance)
+            private void HandlePossibleEscapingOperation(IOperation escapingOperation, ImmutableHashSet<AbstractLocation> escapedLocations)
             {
-                Debug.Assert(escapingOperation != null);
```

Comment:
Majority of code tracking abstract locations/allocations that escaped the method body was moved from Dispose analysis to PointsTo analysis. This was required as we need to mark all the child tuple elements of escaped tuples as escaped, which needs PointsTo data. Additionally, this information about which creation locations escape a method seem to be generic enough to be useful outside dispose analysis, so it made sense to move it to the PointsToAnalysisResult (which is now a new type).